ipaserver/plugins/user.py

```diff
@@ -603,11 +603,14 @@ def post_callback(self, ldap, dn, entry_attrs, *keys, **options):
             if not options.get('all', False):
                 desc_attr = ldap.get_entry(dn, ['description'])
                 entry_attrs.update(desc_attr)
+                entry_attrs._orig['description'] = list(
```

Comment:
How about to move this whole block 20 lines below, where `newentry = ldap.get_entry(dn, ['*'])` is. IIUC that call will download the whole user_entry, so you can just delete `NO_UPG_MAGIC` if needed and you even save one more LDAP operation without ugly hacking of LDAPEntry.